test/gold/tools/getReceipts/api_tests/getReceipts_bad_hash_msg.txt

```diff
@@ -1,4 +1,3 @@
 receipts?transactions=0x060e4cf9fa8d34a8b423b5b3691b2541255ff7974ff16699e104edcfb63bd521%200x0766ec95bf12f9600afc6c33c287f9d2356d85f15f0d7eeeb6647f5d4ddfbc11%200x060e4cf9fa8d34a8b423b5b3691b2541255ff7974ff16699e104edcfb63bd521&fmt=csv
 blockNumber,transactionIndex,transactionHash,status,gasUsed,logsCnt,isError
 1001001,3,0x060e4cf9fa8d34a8b423b5b3691b2541255ff7974ff16699e104edcfb63bd521,1,21000,0,false
-1001001,3,0x060e4cf9fa8d34a8b423b5b3691b2541255ff7974ff16699e104edcfb63bd521,1,21000,0,false
```

Comment:
Note that the new code removes duplicate entries from the user's command line since we put it into a map.